Alpha

```diff
@@ -1,13 +1,14 @@
 {
 	"project": "Alpha",
 	"tags": [
-		"TJ Papove", "Dog Skull 01", "Skully the Spray Can", "WorldBicycleReliefDonator01", "JHBGreenFire", "Hump Day #1"
+		"Alpha", "TJ Papove", "Dog Skull 01", "Skully the Spray Can", "WorldBicycleReliefDonator01", "JHBGreenFire", "Hump Day #1", "Bird Skull 01"
```

Comment:
can't have the tag alpha, too common :/

Reply:
Hey sorry I'm still learning how to use github here. 

"Alpha" wasn't in the last changes and is my business name. 

If you need me to take it out now let me know how you would like me to go about doing it. Do I need to make a new pull request, or can you just be deleted?

Thanks
TJ

Reply:
Hey Stale, 
I made the change. Let me know if its all good, thanks man.